mentorship ios/ViewModels/TaskCommentsViewModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  TaskCommentsViewModel.swift
+//  Created on 28/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import Combine
+
+class TaskCommentsViewModel: ObservableObject {
+    @Published var taskCommentsResponse = [TaskCommentsModel.TaskCommentsResponse]()
+    @Published var newComment = TaskCommentsModel.PostCommentUploadData(comment: "")
+    @Published var postCommentResponse = TaskCommentsModel.PostCommentResponse(message: "", success: false)
+    @Published var isLoading = false
+    @Published var showingEarlier = false
+    @Published var showErrorAlert = false
```

Comment:
nit: Should add comments about what these variables are for esp. `showingEarlier`

Reply:
Done